robottelo/ui/container.py

```diff
@@ -71,7 +71,22 @@ def create(self, resource_name, name, command, parameter_list):
                 {'main_tab_name': 'Environment', 'name': 'TTY', 'value': True},
             ]
         """
-        self.assign_value(locators['container.resource_name'], resource_name)
+        # send_keys() can't send left parenthesis (see SeleniumHQ/selenium#674)
```

Comment:
Thanks for the clarification I didn't get that by reading the code, poor UI skills.